lib_web/context.ml

```diff
@@ -72,15 +80,19 @@ let template t contents =
   let open Tyxml.Html in
   html_to_string (
     html
-      (head (title (txt site.name)) [
+      (head (title (txt site.name))
+        ((List.map (fun font -> link ~rel:[ `Stylesheet ] ~href:font ()) fonts)
+        @ [
           link ~rel:[ `Stylesheet ] ~href:"/css/style.css" ();
-          meta ~a:[a_charset "UTF-8"] ();
-        ]
+          link ~rel:[ `Icon ] ~href:img_ocaml_logo ();
+          meta ~a:[ a_charset "UTF-8" ] ();
+        ])
       )
       (body [
           nav [
+            img ~alt:"" ~src:img_ocaml_logo ~a:[ a_height 29 ] ();
+            div [code [txt site.name]];
             ul (
-              li [a ~a:[a_href "/"] [txt site.name]] ::
```

Comment:
Normally, sites make the top-left logo/site-name go to the home page. If having two links is a problem, I suggest getting rid of the "Home" link instead.

Reply:
I've put this link back.